my-sms/app/models/institute.rb

```diff
@@ -4,4 +4,6 @@ class Institute < ActiveRecord::Base
   attr_accessible :name
 
   validates :name, presence: true
+
+  DEFAULT_PER_PAGE = 10
```

Comment:
So this is the same value for Student and Coursel and most likely any new list page that will be added.
Points more towards one global constant.
How could you refactor this ?

Reply:
I moved it to the application controller in 9e71a9ac8164c8919c65e335950a4b9e97092660 . Its additionally a better fit because the rest of the pagination is handled in the controller, and MVC wise it shouldn't really be part of the model. 

Decorators are arguably an even better fit, but I opted for the application controller as decorators are designed to act on model instances, whereas this is class level behaviour.